### PR TITLE
map_header: Minor improvements

### PR DESCRIPTION
**Pull request by tgsm**
I stole a couple of function names from pokeplatinum.

I recommend reviewing commit-by-commit.

### CODE REVIEW

**Comment by red031000** [comment]
I hope this doesn't conflict with my WIP map renaming PR...

File: asm/overlay_01_021E90C0.s [hunk 1]
```diff
@@ -2181,7 +2181,7 @@ ov01_021EA1F4: ; 0x021EA1F4
 	ldr r0, [r0, #0x20]
 	add r4, r1, #0
 	ldr r0, [r0]
-	bl sub_0203B58C
+	bl MapHeader_IsBuilding
```
Comment (adrienntindall) [inline asm/overlay_01_021E90C0.s:2184]: MapHeader_IsBuilding -> MapHeader_IsInBuilding

File: src/map_header.c [hunk 2]
```diff
@@ -46,200 +46,200 @@ static const u16 sPokemonCenterMaps[] = {
...
+    mapId = MapNumberBoundsCheck(mapId);
+    return sMapHeaders[mapId].areaIcon;
 }
 
-u8 MapHeader_GetMomCallIntroParam(u32 map_no) {
-    map_no = MapNumberBoundsCheck(map_no);
-    return sMapHeaders[map_no].unk_12_C;
+u8 MapHeader_GetMomCallIntroParam(u32 mapId) {
+    mapId = MapNumberBoundsCheck(mapId);
+    return sMapHeaders[mapId].unk_12_C;
```
Comment (adrienntindall) [inline src/map_header.c:125]: unk_12_C ->momCallIntroParam or similar

File: src/map_header.c [hunk 3]
```diff
@@ -46,200 +46,200 @@ static const u16 sPokemonCenterMaps[] = {
...
 }
 
-BOOL MapHeader_IsKanto(u32 map_no) {
-    map_no = MapNumberBoundsCheck(map_no);
-    return sMapHeaders[map_no].is_kanto;
+BOOL MapHeader_IsKanto(u32 mapId) {
```
Comment (adrienntindall) [inline src/map_header.c:128]: IsKanto -> IsInKanto?

**Review by adrienntindall — CHANGES_REQUESTED** [review]
I will simply trust that the data file too big to review non locally is fine and just a ton of variable changes

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**